### DISCUSSION TITLE
support bun.sh

### DISCUSSION THREAD

**Discussion by deadcoder0904** [coollabsio/coolify#1543] [Feature Requests]
pnpm is great but bun is fast.

sometimes i face issue & i have to wait ~2 mins to see if i face another error then fix it, schedule another deploy. it all adds up.

how about having a first-class support for bun.sh?

it works on simpler apps & does it really fast. i wanna use it on my repo as mine is very simple.

i installed it on the server & tried it but nixpack gave issues. would love for it to be supported now that it works on so many projects.

also could do something like this: https://andrekoenig.de/articles/using-bun-as-the-package-manager-in-production-ready-docker-images

**Comment by andrasbacsai** [comment]
Nixpacks supports bun, so it is already supported!
